doc/deployment_guide.md

```diff
@@ -1,29 +1,29 @@
-
-<img src="media/dc99730255437cdd115b74c45cc6c19d.jpg" alt="drawing" width="200px"/>
-
-# Dell EMC Red Hat Ready Architecture Deployment Guide
-
-**Version 13.0**
-
-**Dell EMC Service Provider Solutions**
-
-
-
-
-<div style="page-break-after: always;"></div>
-
-# Trademarks
-Copyright © 2014-2019 Dell Inc. or its subsidiaries. All rights reserved.
-
-Microsoft® and Windows® are registered trademarks of Microsoft Corporation in the United States and/or other countries.
-
-Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered trademarks of Red Hat, Inc., registered in the U.S. and other countries. Linux® is the registered trademark of Linus Torvalds in the U.S. and other countries. Oracle® and Java® are registered trademarks of Oracle Corporation and/or its affiliates.
-
-> DISCLAIMER: The OpenStack® Word Mark and OpenStack Logo are either registered trademarks/ service marks or trademarks/service marks of the OpenStack Foundation, in the United States and other countries, and are used with the OpenStack Foundation's permission. We are not affiliated with, endorsed or sponsored by the OpenStack Foundation or the OpenStack community.
-
-<div style="page-break-after: always;"></div>
-
-# Contents:
+
+<img src="media/dc99730255437cdd115b74c45cc6c19d.jpg" alt="drawing" width="200px"/>
+
+# Dell EMC Red Hat Ready Architecture Deployment Guide
+
+**Version 13.2**
+
+**Dell EMC Service Provider Solutions**
+
+
+
+
+<div style="page-break-after: always;"></div>
+
+# Trademarks
+Copyright © 2014-2019 Dell Inc. or its subsidiaries. All rights reserved.
+
+Microsoft® and Windows® are registered trademarks of Microsoft Corporation in the United States and/or other countries.
```

Comment:
This whole section needs to be updated.  It should match exactly what is in the 13.2 RA. Delete Microsoft, Red Hat, etc and add the others.